server/context.go

```diff
@@ -351,20 +351,22 @@ func (ctx *Context) PGURL(user string) (*url.URL, error) {
 	} else {
 		options.Add("sslmode", "verify-full")
 		requiredFlags := []struct {
-			name  string
-			value *string
+			name     string
+			value    *string
+			flagName string
 		}{
-			{"sslcert", &ctx.SSLCert},
-			{"sslkey", &ctx.SSLCertKey},
-			{"sslrootcert", &ctx.SSLCA},
+			{"sslcert", &ctx.SSLCert, "cert"},
+			{"sslkey", &ctx.SSLCertKey, "key"},
+			{"sslrootcert", &ctx.SSLCA, "ca-cert"},
```

Comment:
@mberhault what was the command you used to produce a crash? I think the issue is the new `insecureValue` type, which we should be able to special case pretty easily. I'll take a look.

In terms of this change, I'm fine with the safer `GetFlagName` method to prevent crashing, but I'd rather not have a default name. There's really no point in having the map if we are just going to duplicate the flag name everywhere anyway.